### PR TITLE
Added Google Drive Realtime API Element

### PR DESCRIPTION
**Pull request by sethhoward1988**

### CODE REVIEW

**Comment by ebidel** [comment]
cc @nicolasgarnier

File: google-realtime-api.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!--
+Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
```
Comment (ebidel) [inline google-realtime-api.html]: You can remove this header.

**Comment by addyosmani** [comment]
Great work, Seth! This LGTM. Want to update the Realtime Playground to start using this once it lands?

**Comment by sethhoward1988** [comment]
Yes, I'm planning on putting this into the playground soon.

On Sat, Jan 24, 2015 at 1:40 PM, Addy Osmani <notifications@github.com>
wrote:

> Merged #38 <https://github.com/GoogleWebComponents/google-apis/pull/38>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleWebComponents/google-apis/pull/38#event-224209067>
> .
>



-- 
Seth

**Comment by addyosmani** [comment]
:+1:

**Pull request merged**